catroid/src/org/catrobat/catroid/ProjectManager.java

```diff
@@ -444,6 +444,7 @@ public int getCurrentSpritePosition() {
 		return project.getSpriteList().indexOf(currentSprite);
 	}
 
+	//Function apperantly never used please check if necessary!
```

Comment:
The [code inspection for 'Unused declaration'](http://stackoverflow.com/a/6587932/2774399) will tell you another 241 unused items. :wink:
Please remove this comment. Feel free to create a Jira ticket to remove unused code. Beware, some code is used by reflection but may be reported as well.
I also would :heart: to see the IntelliJ Code Inspection tool running on Jenkins [CAT-1672](https://jira.catrob.at/browse/CAT-1672)